frontend/src/app/shared/export-item-modal/export-item-modal.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'oib-export-item-modal',
+  templateUrl: './export-item-modal.component.html',
+  styleUrl: './export-item-modal.component.scss',
+  imports: [],
+  standalone: true
+})
+export class ExportItemModalComponent {
+  delimiterOptions: Array<string> = [',', ';'];
+  selectedDelimiter: string = ',';
+
+  constructor(private modal: NgbActiveModal) {}
+
+  save() {
+    const delimiter = document.querySelector('select')?.value;
```

Comment:
you should use a form control field (see other forms inputs of type select). Also, it should be required, so it's not useful to check if it exists